lib/Dialect/TTNN/Analysis/LegalGridAnalysis.cpp

```diff
@@ -11,6 +11,16 @@ namespace mlir::tt::ttnn {
 
 bool mock_is_output_tensor_legal_for_op(Operation *op, tt::LayoutAttr layout) {
   // Placeholder, needs to be replaced with a call the the TTNN op interface.
+
```

Comment:
nit: We shouldn't commit commented code, maybe we can capture this with an issue and add a TODO comment here with issue id?

Reply:
true, the purpose of leaving them was to showcase on how it's going to look like, otherwise it might be too abstract.

Reply:
removed commented out code.